src/Dfe.PlanTech.Infrastructure.Data/CmsDbContext.cs

```diff
@@ -94,13 +145,36 @@ public CmsDbContext(DbContextOptions<CmsDbContext> options) : base(options)
 
     protected override void OnModelCreating(ModelBuilder modelBuilder)
     {
+        modelBuilder.ApplyConfiguration(new RecommendationSectionEntityTypeConfiguration());
```

Comment:
Dunno where it's gone (merge issue?) but:

- You should be able to replace these two `ApplyConfiguration`s with:
`modelBuilder.ApplyConfigurationsFromAssembly(typeof(CmsDbContext).Assembly);`

- You should then be able to also remove the configuration in this method for the `CategoryDbEntity`, `PageDbEntity` and `SectionDbEntity.`